articles/multifactor-authentication/guardian/admin-guide.md

```diff
@@ -40,6 +40,28 @@ When your users sign up with SMS they enter their phone number's country code an
 
 After sign up they receive a six digit code to their phone.  They need to enter this code into the box, and then they will get a recovery code. They will need this code to login if you do not have their device. You may be contacted if they have lost their recovery code and their device, then you will need to [reset the user's MFA](/admin-guide#reset-mfa-for-a-user).
 
+### Configuring Guardian SMS with Twilio
+
+When initially setting up SMS, you have up to 100 SMS and up to 10 SMS/Hour per user to test Guardian. This limit can be removed by setting up a Twilio account to use. Click on the **SMS** box to configure your SMS settings.
```

Comment:
We might want to mention that the 100 SMS are only for trial purposes and that you are supposed to set up your own credentials in a productive app. 

Regarding the limit of 10 SMS/Hour it is to protect customer against abusive usage, so it won't be removed once you set up your credentials [the copy on dashboard is wrong, my bad.] (we might want to mention the reason for this limit to exist)